docs/_docs/automatic-layout-containers.md

```diff
@@ -23,32 +23,32 @@ Both nodes and layoutSpecs conform to the `<ASLayoutable>` protocol.  Any `ASLay
     <th>Description</th> 
   </tr>
   <tr>
-    <td><b><code>ASInsetLayoutSpec</code></b></td>
+    <td><b>`ASInsetLayoutSpec`</b></td>
```

Comment:
Revert all of these modifications within the table. Code snippets inside HTML tags, must use \<code\>\</code\>  rather than \`\`